modules/imap/output_modules.php

```diff
@@ -243,7 +243,7 @@ protected function output() {
                     $txt .= ' | <a class="reply_all_link hlink" href="?page=compose&amp;reply_all=1'.$reply_args.'">'.$this->trans('Reply-all').'</a>';
                 }
                 else {
-                    $txt .= ' | <a class="reply_all_link hlink disabled_link">'.$this->trans('Reply-all').'</a>';
+                    $txt .= ' | <a class="reply_all_link hlink" href="?page=compose&amp;reply_all=1'.$reply_args.'">'.$this->trans('Reply-all').'</a>';
```

Comment:
It looks good, have you checked if the code below can add the `disabled_link` class again without you wanting to?

https://github.com/jasonmunro/cypht/blob/9a61e208fbd462c02dc561bb2db6625ea4e7d353/modules/imap/site.js#L721-L724

Reply:
It can only add the `disabled_link` when the page has not yet loaded

Reply:
Good, I just realized that the `else` below is unnecessary:

https://github.com/jasonmunro/cypht/blob/8aadcbfe1cde41043c571af96992ba7f62670fd7/modules/imap/output_modules.php#L242-L247

The code is the same for both conditions

Reply:
I didn't pay attention to that, let me remove one and the condition as it is no longer in use